webextensions/api/bookmarks.json

```
@@ -205,7 +205,7 @@
             "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/bookmarks/getRecent",
             "support": {
               "chrome": {
-                "version_added": "≤58"
+                "version_added": "4"
```

:x: https://github.com/chromium/chromium/commit/a3c94c713dcf12e7d95b7c4548137d7beb790e9c in [4.0.276.0](https://github.com/chromium/chromium/releases/tag/4.0.276.0) AFTER 4.0.223

```suggestion
                "version_added": "5"
```